src/main/java/eu/dissco/core/digitalspecimenprocessor/service/ProcessingService.java

```diff
@@ -348,6 +375,50 @@ private Set<DigitalSpecimenRecord> createNewDigitalSpecimen(List<DigitalSpecimen
     }
   }
 
+  private void gatherDigitalMediaObjectForNewRecords(
+      Map<DigitalSpecimenRecord, Pair<List<String>, List<DigitalMediaObjectEventWithoutDoi>>> digitalSpecimenRecords) {
+    log.info("Publishing digital media object events for processing");
```

Comment:
Mainly because that is the structure used in the above method. It didn't make sense to me do loop over the list there to extract the part that we don't use and then loop over it again when publish the media.